lib/system/api/containers.toit

```diff
@@ -39,6 +39,20 @@ interface ContainerService:
   image-writer-commit handle/int flags/int data/int -> uuid.Uuid
   static IMAGE-WRITER-COMMIT-INDEX /int ::= 5
 
+  // INDEX 8 is reserved for ContainerMessageService so that
```

Comment:
Not sure why this is necessary? A single provider can provide multiple independent API implementations.

Reply:
I thought it would be easier if a single class can implement both. It's, after all, the typical use case.